release.yaml

```diff
@@ -44,6 +44,7 @@ rancher-vsphere-csi:
 - 100.1.1+up2.3.0-rancher1
 rancher-vsphere-cpi:
 - 100.1.1+up1.0.1
```

Comment:
Please remove the 100.1.1+up1.0.1 entry and its assets, 100.2.0+up1.1.0 replaces it.

Reply:
Okay. I made removed the chart entries, can you please review it again?